docs/content/operator/full-node/troubleshooting.mdx

```diff
@@ -105,11 +104,18 @@ If you continue to see issues, do the following steps:
 
       Run the respective prepare script again depending on your network
       <Tabs groupId="network" queryString>
+          <TabItem label="Mainnet" value="mainnet">
+            ```shell
+            ./prepare_mainnet.sh
```

Comment:
I think all the scripts are now called "prepare.sh" by the workflow, so I guess we don't need the network switcher here anymore?